docs/Processes.md

```diff
@@ -221,8 +228,69 @@ the gateway api. Use .api.p“.gw.\*” for more details.
 | .gw.asyncexecjpt\[query; servertypes; joinfunction; postback; timeout\] | Execute the specified query against the required list of servers. Use the specified join function to aggregate the results. If the postback function is not set, the client must block and wait for the results. If it is set, the result will be wrapped in the specified postback function and returned asynchronously to the client. The query will be timed out if the timeout value is exceeded. |
 
 
-For the purposes of demonstration, assume that the queries must be run
-across an RDB and HDB process, and the gateway has one RDB and two HDB
+### Client Call Examples
+
+Here are a couple of examples for using client calls via a handle to the gateway process. 
```

Comment:
There are more than a couple 
I think it should be re-emphasized here that the async calls are only relevant for users on kdb < 3.6. For 3.6+ just use sync